### PR TITLE
[5.2] Copy to clipboard for Android version 9 and below

### PR DESCRIPTION
**Pull request by blackdevelopa**
**Description**

This PR should prevents android versions less than ten from copying SRP to clipboard  

**Checklist**

* [ ] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

**Issue**

Progresses https://github.com/MetaMask/mobile-planning/issues/243

### CODE REVIEW

File: app/components/Views/RevealPrivateCredential/index.js [hunk 1]
```diff
@@ -438,6 +438,10 @@ class RevealPrivateCredential extends PureComponent {
   renderTabView(privateCredentialName) {
     const { clipboardPrivateCredential } = this.state;
     const { styles, colors, themeAppearance } = this.getStyles();
+    const leastAndroidSupportedVersion = 10;
```
Comment (sethkfman) [inline app/components/Views/RevealPrivateCredential/index.js:441]: I would place this inside `AppConstants.js`

**Review by sethkfman — APPROVED** [review]
LGTM

**Comment by plasmacorral** [comment]
I am able to copy both private key and SRP running android 6.

**Comment by plasmacorral** [comment]
Able to copy values in Android 10, 11, 12 and iOS.  
Copy button is not available in Android versions 6, 7, 8, 9.

**Review by gantunesr — APPROVED** [review]
LGTM

**Pull request merged**